### PR TITLE
Add a simple reflector allowing POJO models to be serialized

### PR DESCRIPTION
**Pull request by praxxis**

### CODE REVIEW

**Comment by tomdale** [comment]
@praxxis What is the use case?

**Comment by praxxis** [comment]
@tomdale in my specific case, a [model](https://github.com/monegraph/mcast-api/pull/28/files#diff-44896ec8bc5956f273484819c20e5e76) which is not persisted, and doesn't have the concept of "all ID's" or relationships. Some sort of reflector is needed to [serialize said model](https://github.com/monegraph/mcast-api/pull/28/files#diff-c29efab34ebe40cbea56a499cc2d621cR52).

**Comment by praxxis** [comment]
@tomdale fwiw if you're opposed to having this in core I have no problem pulling it back into our API.

File: packages/starspot-json-api/src/serializer.ts [hunk 1]
```diff
@@ -71,11 +71,11 @@ class Serializer {
     this.primaryResources.push(...result.resources);
   }
 
-  async serialize(model: any, included: Included): Promise<JSONAPI.DataDocument> {
+  async serialize(model: any, included: Included = new Included()): Promise<JSONAPI.DataDocument> {
```
Comment (tomdale) [inline packages/starspot-json-api/src/serializer.ts]: Does TypeScript automatically infer the type if you assign it? In other words, can this be condensed to just: `included = new Included()`?

Comment (praxxis) [reply packages/starspot-json-api/src/serializer.ts]: indeed it can be

**Pull request merged**